biz.aQute.bndlib.tests/test/test/MacroTest.java

```diff
@@ -1780,6 +1781,14 @@ public void testBase64() {
 		assertEquals(b64, b.getProperty("b64"));
 	}
 
+	@Test
+	public void testBndVersion() {
+		Processor b = new Processor();
+		b.setProperty("thebndversion", "${bndversion}");
+		String version = b.getProperty("thebndversion");
+		assertTrue(Version.isVersion(version));
```

Comment:
This does not test that the macro returns the current bnd version. You need to compare with `About.CURRENT` which is the current bnd version.